docs/specification.md

```diff
@@ -140,8 +132,35 @@ inherent:
   - sequences
 ```
 
-This schema is the *seqcol schema*, and sequence collection objects in this structure are said to be the *canonical seqcol object representation*.
-Here's an example of a sequence collection organized into the canonical seqcol object representation:
+This example schema is the minimal standard schema. We RECOMMEND that all implementations use this as a base schema, adding additional attributes as needed, but *without changing the inherent attributes list*, because this will keep the identifiers compatible across implementations.
```

Comment:
Do we want to add "accessions" into the minimal schema, or is that rather meant as an example?

Reply:
One could for instance argue that one would want a different array for each type of accession number. I cannot recall having this discussion (but I might also have been absent)?

Reply:
Also, since `sorted_name_length_pairs` and `sorted_sequences` are described in the specification itself, shouldn't we provide a template for how these attributes should be added to the schema, if used?